newsletter-july-2022.rst

```diff
@@ -0,0 +1,62 @@
+.. post:: July 12, 2022
+   :tags: newsletter, python
+   :author: Eric Holscher
+   :location: BND
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - July 2022
+====================================
+
+Summer has come,
+which means our overall development has slowed a bit as the team takes some well-deserved vacation time.
+That said,
+we're still excited about our `recent hire`_ and the ongoing work we've been doing to support the documentation ecosystem.
+
+Our focus for Q3 (July-Sept) of 2022 is around improving our frontend and marketing pages. 
+This includes a fancy new marketing website, 
+as well as a revamped dashboard UX that will make many features nicer for our users.
+
+.. _recent hire: https://github.com/benjaoming
+
+
+New features
```

Comment:
Not sure if they are _features_, but at least they are bugfixes.

- sphinx-hoverxref works nicely with Jupyter Book now (https://github.com/executablebooks/sphinx-book-theme/issues/577)
- manual references fixed on sphinx-hoverxref (https://github.com/readthedocs/sphinx-hoverxref/issues/199)
- on development, we are using `nodemon` now to watch file changes --making builders to restart automatically without issues

Reply:
@humitos we definitely don't want to include dev changes here. I think the hoverxref stuff is worth mentioning.